src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -606,13 +656,13 @@ private TypeSymbol BindNamedAttributeArgumentType(AttributeArgumentSyntax namedA
         /// CONSIDER: Can we share some code will call rewriting in the local rewriter?
         /// </remarks>
         private ImmutableArray<TypedConstant> GetRewrittenAttributeConstructorArguments(
-            out ImmutableArray<int> constructorArgumentsSourceIndices,
             MethodSymbol attributeConstructor,
             ImmutableArray<TypedConstant> constructorArgsArray,
             ImmutableArray<string?> constructorArgumentNamesOpt,
             AttributeSyntax syntax,
             ImmutableArray<int> argumentsToParams,
             BindingDiagnosticBag diagnostics,
+            bool expanded,
```

Comment:
Reviewers should be aware that the diff gets quite hard to read in this file from here on. You might be better off just separately reviewing the lines deleted and lines added due to large gaps in the regular split view.